app/main/posts/posts-routes.js

```diff
@@ -1,155 +1,158 @@
 module.exports = [
     '$stateProvider',
+    '$urlMatcherFactoryProvider',
 function (
-    $stateProvider
+    $stateProvider,
+    $urlMatcherFactoryProvider
 ) {
+    $urlMatcherFactoryProvider.strictMode(false);
 
     $stateProvider
     .state(
         {
-            name: 'list',
+            name: 'list', /** I agree, this is a terrible name for this state. Ideas are welcome.**/
```

Comment:
I'd suggest just using `posts` so you get `posts.map` and `posts.data`

Reply:
ok sweet, IDK why I didn't think of that.